src/components/Header/MobileMenu.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from "react"
+import { useSelector, useDispatch } from "react-redux"
+import { NavLink } from 'react-router-dom'
+
+import { fetchLogout } from "../../ducks/userInfo"
+import LogoHead from "./LogoHead"
+import PointsBal from "./PointsBal"
+import Navigation from "./Navigation"
+import {
+    ContainerHeader,
+    ExitIconWrapper,
+    LeftBlock,
+    MenuHeaderBlock,
+    MenuInitiaLetterCircle,
+    MenuInitiaLetter,
+    MenuName,
+    MobileHamburgerContainer,
+    MobileHamburgerLine,
+    MobileHamburgerbackground,
+    MobileHamburgerLoginBox
+} from "./Header.styled"
+import ExitIconMobile from "../../assets/image/Header-exit-icon-mobile.svg"
+
+const MobileMenu = () => {
+    let userName = 'email'
+    const dispatch = useDispatch()
+    const logOut = () => dispatch(fetchLogout(formData))
+    const userData = useSelector(state => state.userInfo.userData)
+    if (userData) {
+        userName = userData.email
+    }
+    const nameLetter = userName && (userName.charAt(0))
+    const [isOpen, setIsOpen] = useState(false)
+    const toggle = () => { setIsOpen(!isOpen) }
+    const childToggle = () => { setIsOpen(!isOpen) }
```

Comment:
As there is no difference between them you can use one function

Reply:
Right now I am using one function. Indeed, they were identical. Thanks for checking))
![image](https://user-images.githubusercontent.com/96341034/189454795-ba51832e-0de6-40a6-a443-9f6cdf534d20.png)